src/functions.js

```diff
@@ -1,51 +1,78 @@
+};
 
-export const getFileData = (fileDirectory) => {
-    return fs.readFileSync(fileDirectory);
-}
+export const getFileData = fileDirectory => {
+  return fs.readFileSync(fileDirectory);
+};
 
 export const exportSvg = (svgo, data, fileName) => {
-    svgo.optimize(data)
+  svgo
+    .optimize(data)
     .then(result => {
-        let exportDirectory = EXPORT_DIRECTORY + fileName;
+      let exportDirectory = EXPORT_DIRECTORY + fileName;
 
-        tools.ExportSVG(result.data, exportDirectory)
+      tools
+        .ExportSVG(result.data, exportDirectory)
         .then(() => console.log(`Exported: ${fileName}`))
         .catch(err => console.log(err));
     })
     .catch(err => console.log(err));
-}
+};
 
 export const buildReport = directory => {
-    console.log(`\ndirectory: ${directory}`);
-    const fileNames = getAllFileNames(directory);
-    fileNames.forEach(fileName=> {
-        const filePath = directory + fileName;
-        const stat = fs.statSync(filePath)
-        console.log(`${fileName}  ${stat.size}`)
-    })
-}
+  console.log(`\ndirectory: ${directory}`);
+  const fileNames = getAllFileNames(directory);
+  fileNames.forEach(fileName => {
+    const filePath = directory + fileName;
+    const stat = fs.statSync(filePath);
+    console.log(`${fileName}  ${stat.size}`);
+  });
+};
 
 export const comparisonReport = (originalDirectory, optimizedDirectory) => {
-    console.log(`${originalDirectory} ->  ${optimizedDirectory}`);
-    const fileNames = getAllFileNames(originalDirectory);
-    fileNames.forEach(fileName => {
-        const origionalFilePath = originalDirectory + fileName;
-        const optimizedFilePath = optimizedDirectory + fileName;
-        const origionalSize = fs.statSync(origionalFilePath).size;
-        const optimizedSize = fs.statSync(optimizedFilePath).size;
-        console.log(`${fileName} :  ${origionalSize} -> ${optimizedSize}`);
-    })
-}
+  console.log(`${originalDirectory} ->  ${optimizedDirectory}`);
+  const fileNames = getAllFileNames(originalDirectory);
+  fileNames.forEach(fileName => {
+    const originalFilePath = originalDirectory + fileName;
+    const optimizedFilePath = optimizedDirectory + fileName;
+    const originalSize = fs.statSync(origionalFilePath).size;
+    const optimizedSize = fs.statSync(optimizedFilePath).size;
+    console.log(`${fileName} :  ${origionalSize} -> ${optimizedSize}`);
+  });
+};
+
+export const writeSvgCodeIntoSvgFiles = directory => {
+  const fileNames = getAllFileNames(directory);
+  fileNames.forEach(fileName => {
+    var filePath = directory + fileName;
+    var oldData = getFileData(filePath).toString();
+    if (oldData.search(CODE_ELEMENT) < 0) {
+      var PositionToInsert = oldData.indexOf(">")+1;
+      var newData = [
+        oldData.slice(0, PositionToInsert),
+        CODE_ELEMENT,
+        oldData.slice(PositionToInsert, oldData.length)
+      ].join("");
+
+      fs.writeFile(filePath,newData, err => {
+        if (!err) {
```

Comment:
err is probably a string, switch your logic around, if(err) checks if err exists, then is handled accordingly, please stick to this way of handling errors